core/templates/components/question-directives/modal-templates/question-editor-modal.directive.html

```diff
@@ -28,12 +28,12 @@ <h4 ng-if="!newQuestionIsBeingCreated" style="margin-top: 4vh;"> <strong> Questi
                    can-edit-question="canEditQuestion"
                    question-changed="questionChanged">
   </question-editor>
-  <div class="alert alert-danger" ng-if="validationError">
-    <[validationError]>
+  <div class="alert alert-danger" ng-if="validationError || (question.getStateData().interaction.id && isSaveButtonDisabled())">
+    <angular-html-bind html-data="validationError || isSaveButtonDisabled()"></angular-html-bind>
```

Comment:
I don't quite follow this. Why are you binding html-data to a value that might be equal to the result of isSaveButtonDisabled()? Also why does it need to be html (why can't it be plaintext)?

Reply:
isSaveButtonDisabled() runs the valiation and returns the validation error. I will rename the function so that it is easy to understand what it does. Thanks!

Reply:
Also it has to be html because the validation message contains <li>/<ul>

Reply:
See a [later comment](https://github.com/oppia/oppia/pull/9337#discussion_r433650401) suggesting how to handle that -- in general, you don't want to pass html strings around.

Reply:
Changed this a bit.